rugby.ipynb

```diff
@@ -0,0 +1,3579 @@
+{
```

Comment:
<p>As you can see when exploring the dimensions and coordinates of the generated object, they are not the same as the example available from within ArviZ. If you explore all the groups, you will see that only 4 dimensions are present: chain, draw, team and match. In the idata generated here, several more are included. The usage of coords and dims in ArviZ is the following:</p><p> </p><ul><li>coords is a dictionary whose name are dimension names (thus team and match will be keys here, chain and draw are added by ArviZ so don't need to be here) and whose values are coordinate values. The dict you currently have is already correct regarding the team dimension, only the match one is missing.</li><li>dims is a dictionary whose values are variable names and whose values are lists of the dimensions for each variable. Here, the keys are theta and obs. theta and obs where variable names in the 8 schools model, but they are not here. Therefore, they are ignored and the team dimension is not added anywhere. The variables that should be added are the ones who have either team/match dimension in the arviz idata example. </li></ul><p> </p><p>Here is the print of the posterior group for the example in arviz:</p><p> </p><pre class="ql-syntax" spellcheck="false">&lt;xarray.Dataset&gt;
Dimensions:&nbsp;&nbsp;&nbsp; (chain: 4, draw: 500, team: 6)
Coordinates:
&nbsp; * chain&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (chain) int64 0 1 2 3
&nbsp; * draw&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (draw) int64 0 1 2 3 4 5 6 7 ... 492 493 494 495 496 497 498 499
&nbsp; * team&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (team) object 'Wales' 'France' 'Ireland' ... 'Italy' 'England'
Data variables:
&nbsp;&nbsp;&nbsp; home&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (chain, draw) float64 ...
&nbsp;&nbsp;&nbsp; intercept&nbsp; (chain, draw) float64 ...
&nbsp;&nbsp;&nbsp; atts_star&nbsp; (chain, draw, team) float64 ...
&nbsp;&nbsp;&nbsp; defs_star&nbsp; (chain, draw, team) float64 ...
&nbsp;&nbsp;&nbsp; sd_att&nbsp;&nbsp;&nbsp;&nbsp; (chain, draw) float64 ...
&nbsp;&nbsp;&nbsp; sd_def&nbsp;&nbsp;&nbsp;&nbsp; (chain, draw) float64 ...
&nbsp;&nbsp;&nbsp; atts&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (chain, draw, team) float64 ...
&nbsp;&nbsp;&nbsp; defs&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (chain, draw, team) float64 ...
</pre><p> </p><p>As you can see, atts and defs (as well as their star versions) have <code>team</code>  as dimension, thus, these variables should be keys in the <code>dims</code>  dict</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_example_data/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<ol><li><strong>coords</strong>: I am little confused with what value must be mapped to the key "match"? Could you please help me in identifying the same.</li><li><strong>dims</strong>: I have modified the dims argument. Following is the new code:</li></ol><p>							<code>dims={"atts": teams, "defs": teams, "atts_star": teams, "defs_star": teams}</code>  </p><p> </p><p>Is this as per the requirement for the dims?</p><p> </p><p>Thank You!</p>

Reply:
1. To generate the same coordinate values for match as the ones in the current ArviZ rugby example you can do the following:

```
match = [f"{home} {away}" for home, away in zip(df.home_team, df.away_team)]
```

2. Note that `dims` has to be a dictionary of lists, thus, it should be `dims={"atts": [teams], "defs": [teams], "atts_star": [teams], "defs_star": [teams]}`

Reply:
<p>Thank You @OriolAbril for your help and nice explanation!</p><p> </p><p>I have made the required changes for this section of the code. </p>